packages/ketcher-react/src/index.tsx

```diff
@@ -31,9 +30,9 @@ function Editor(props: EditorProps) {
       ...props,
       element: rootElRef.current
     })
-  }, [])
+  }, [props])
```

Comment:
I don't think, that we should add props to dependencies, because in this case `init` function can be called a few times. and it can broke something, isn't it?

Reply:
comments added